### ISSUE TITLE
Selecting bearers also selects other tool-wielding settlers

### ISSUE THREAD

**Issue by tehKaiN** [jsettlers/settlers-remake#772]
As I understand recruiting specialists via selecting bearers is a new feature made for convenience. It's cool and I don't have a problem with it.

Unfortunately, when selecting bearers, other tool-wielding settlers also become selected (stone cutter etc.), although they can't be converted into any specialist. So what's the intended behavior? Ability to convert tool-wielder to specialist and making it drop his tool, or should they not be selectable at all?

**Comment by paulwedeck** [comment]
You can select only settlers of one specific type by double clicking on one.

**Comment by tehKaiN** [comment]
Yes, I understand, but is selecting tool-wielding workers is a bug or incomplete feature since you can't really do anything with them when selected?

**Comment by paulwedeck** [comment]
@andreas-eberle
